datasets.py

```diff
@@ -255,8 +258,9 @@ def from_path(self, file_path):
 
         #Note: bbox digits should be: x, y, width, height. all numbers are scaled to be between 0 and 1
         person_bbox = None # optional
+        #Note: gender should be in a list because attribute_based assumes each image can have more than 1 value
```

Comment:
this comment still isn't very clear to me, maybe given a small example like "[attribute1, attribute2], [bbox1, bbox2]", or something